extensions/sonarcloud/templates/gradle/template.json

```diff
@@ -0,0 +1,109 @@
+{
+  "id": "sonarcloud_gradle_template",
+  "name": "Gradle with SonarCloud",
+  "category": "Build",
+  "iconTaskId": "ce096e50-6155-4de8-8800-4221aaeed4a1",
+  "defaultHostedQueue": null,
+  "description": "Build your Java projects and run tests with Gradle using a Gradle wrapper script, and analyze with SonarCloud.",
+  "template": {
+    "build": [
+      {
+        "environment": {},
+        "enabled": true,
+        "continueOnError": false,
+        "alwaysRun": false,
+        "displayName": "Prepare analysis on SonarCloud",
+        "timeoutInMinutes": 0,
+        "task": {
+          "id": "14d9cde6-c1da-4d55-aa01-2965cd301255",
+          "versionSpec": "1.*",
+          "definitionType": "task"
+        },
+        "inputs": {
+          "scannerMode": "Other",
+          "configMode": "file",
+          "configFile": "sonar-project.properties",
```

Comment:
configMode and configFile are not visible when scannerMode = 'Other'. Do you really have to define a value in the template?